src/init/common.cpp

```diff
@@ -28,6 +29,7 @@ void AddLoggingArgs(ArgsManager& argsman)
         ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-debugexclude=<category>", "Exclude debugging information for a category. Can be used in conjunction with -debug=1 to output debug logs for all categories except the specified category. This option can be specified multiple times to exclude multiple categories.", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
```

Comment:
The semantic of `-debug` and `-debugexclude` seem to have been changed in this PR which warrants an update of their help text.

Before `-debug=net` would enable debug log messages for `net`. Now `-debug=net` enables the `net` category for logging and the severity level is chosen separately via `-loglevel=net:...`.

Reply:
Only for LogPrintLevel messages, of which there are only two currently that have a debug level. I agree this should be done as part of the changes in the parent PR before migrating log messages to LogPrintLevel.

Reply:
> The semantic of `-debug` and `-debugexclude` seem to have been changed in this PR which warrants an update of their help text.
> 
> Before `-debug=net` would enable debug log messages for `net`. Now `-debug=net` enables the `net` category for logging and the severity level is chosen separately via `-loglevel=net:...`.

Good observation! This is only the case for LogPrintLevel messages with a severity level of `debug` or `trace`, of which there are only two currently that are `debug` and none that are `trace`. In other words, `-loglevel` doesn't affect much of anything yet, but these helps should be updated before migrating further log messages to LogPrintLevel debug/trace. Done now here as I'm retouching.